README.md

```diff
@@ -59,7 +59,8 @@ Tested with:
 
 For example, on Ubuntu you can install the dependencies with the following
 command:  
-`apt install cmake libboost-filesystem-dev txt2tags`
+`sudo apt-get install cmake libboost-filesystem-dev txt2tags pkg-config`
+
```

Comment:
Oh, why the addition of this empty line, actually?

Reply:
This makes output in browser better IMO.

Reply:
Because you have removed the double-space at the end of the line, which is the markdown way for line break... :(

Reply:
In my browser it is not working, for example: https://github.com/01org/thunderbolt-software-user-space/tree/f02af41870d25587d2c30426c305d24354ef0154

I see following
'''
For example, on Ubuntu you can install the dependencies with the following command:
apt install cmake libboost-filesystem-dev txt2tags On Fedora, use this:
dnf install cmake boost-devel txt2tags
'''

Reply:
Which is exactly the commit where you removed the double-space as you can see from the diff there...

See on the commit before it:
https://github.com/01org/thunderbolt-software-user-space/tree/0af08b7ff3659e62dd308058c1223789b748a0c3